plugin/pkg/shoot/validator/admission.go

```diff
@@ -951,7 +951,8 @@ func (c *validationContext) validateShootNetworks(a admission.Attributes, worker
 					workerless,
 				)...)
 
-				if c.shoot.Status.Networking != nil {
+				// validate network disjointedness with seed networking if networking status is non-empty
```

Comment:
To be consistent with the upper comment
```suggestion
				// validate network disjointedness with seed networks if networking status is non-empty
```